### ISSUE TITLE
"Maximum call stack size exceeded" occurs when rendering large graph

### ISSUE THREAD

**Issue by only52607** [tintinweb/vscode-interactive-graphviz#164]
![image](https://github.com/tintinweb/vscode-interactive-graphviz/assets/30565026/3adb388d-5556-45c7-957d-af2195388c26)

This is an example dot that causes the error, it contains 5000 nodes.

[large_graph.zip](https://github.com/tintinweb/vscode-interactive-graphviz/files/13808043/large_graph.zip)

**Comment by sneko** [comment]
@only52607 did you find a solution? I have a graph of 30k nodes to plot 😄

**Comment by only52607** [comment]
> @only52607 did you find a solution? I have a graph of 30k nodes to plot 😄

No, I can only use the official executable program to generate images for large graphs.

**Comment by sneko** [comment]
@only52607 I ended doing almost the same thing:
1. I generate a .svg with the executable
2. I open it in Chrome, it's pretty decent and it allows to search for text, to use trackpad to slide and zoom, and from what I remember there is the "hot reload" while remaining at the same position on the image (which is great when it's too wide/tall). I did not find an equivalent tool after multiple testings.

Hope it helps 👍
